app/views/users/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="row user-page">
+  <div class="columns medium-10">
+    <h1>All Users</h1>
+
+    <div class="row">
```

Comment:
You have divs with h tags, but that's it. Are these supposed to be here?

Reply:
There's more in that file. It has a list of all users and the ability to delete those users.